### DISCUSSION TITLE
Wacom Tablet not recognized in Horus

### DISCUSSION THREAD

**Discussion by doobes** [elementary/triage#361] [Q&A] [answered]
I'm getting a bunch of stuff reconnected after a house move.

I've got a Wacom Intuous Pro Small tablet.  I've plugged the dongle into the desktop and actuated the tablet by pressing on one of the buttons, but EOS doe not see the tablet.

Thoughts?

**Comment by hrstwn** [comment]
My experience with Wacom and Linux has been great, it just works out of the box, with pressure and everything.
Does `xinput` recognize the tablet?

**Reply by doobes** [reply]
Negative

It shows up in lsusb though.

chris@Elementary:$ xinput
⎡ Virtual core pointer                    	id=2	[master pointer  (3)]
⎜   ↳ Virtual core XTEST pointer              	id=4	[slave  pointer  (2)]
⎜   ↳ Logitech M310                           	id=9	[slave  pointer  (2)]
⎜   ↳ Logitech K520                           	id=10	[slave  pointer  (2)]
⎣ Virtual core keyboard                   	id=3	[master keyboard (2)]
    ↳ Virtual core XTEST keyboard             	id=5	[slave  keyboard (3)]
    ↳ Power Button                            	id=6	[slave  keyboard (3)]
    ↳ Power Button                            	id=7	[slave  keyboard (3)]
    ↳ Dell WMI hotkeys                        	id=11	[slave  keyboard (3)]
    ↳ UVC Camera (046d:0809)                  	id=8	[slave  keyboard (3)]
    ↳ Logitech K520                           	id=12	[slave  keyboard (3)]
chris@Elementary:$ lsusb
Bus 002 Device 002: ID 8087:8000 Intel Corp. Integrated Rate Matching Hub
Bus 002 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 001 Device 002: ID 8087:8008 Intel Corp. Integrated Rate Matching Hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 003 Device 039: ID 046d:0809 Logitech, Inc. Webcam Pro 9000
Bus 003 Device 037: ID 2109:2812 VIA Labs, Inc. VL812 Hub
Bus 003 Device 035: ID 2109:2812 VIA Labs, Inc. VL812 Hub
Bus 003 Device 038: ID 056a:0084 Wacom Co., Ltd ACK-40401 [Wireless Accessory Kit]
Bus 003 Device 036: ID 046d:c52b Logitech, Inc. Unifying Receiver
Bus 003 Device 034: ID 0b95:6802 ASIX Electronics Corp. AX68002 KVM Switch SoC
Bus 003 Device 033: ID 05e3:0608 Genesys Logic, Inc. Hub
Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub

**Reply by hrstwn** [reply, marked as answer]
Oh, it's wireless?
Can you connect it via USB?

**Comment by doobes** [comment]
LOL.

I didn't even know it was possible to run it direct connected.

For use on my Windows box, I had it plugged into a power supply and used the dongle for connectivity.

Direct connect works fine in either.

Who knew?

Thanks

**Reply by hrstwn** [reply]
I've never tried wireless tablets at all 😅
There probably is someone who have used it wirelessly on Linux but I have no idea. Maybe you'll need to install additional driver.
